### PR TITLE
Feature/offer.edit.synchro

### PR DESCRIPTION
**Pull request by rdrewa**

### CODE REVIEW

File: Volontulo/app/src/main/java/com/stxnext/volontulo/ui/offers/AddOfferFragment.java [hunk 1]
```diff
@@ -286,19 +292,21 @@ public void onResponse(Call<CreateResponse> call, Response<CreateResponse> respo
...
                     realm.close();
                 } else {
-                    Converter<ResponseBody, CreateError> converter = VolontuloApp.retrofit.responseBodyConverter(CreateError.class, new Annotation[0]);
+                    Converter<ResponseBody, SaveError> converter = VolontuloApp.retrofit.responseBodyConverter(SaveError.class, new Annotation[0]);
                     try {
-                        final CreateError error = converter.convert(response.errorBody());
+                        final SaveError error = converter.convert(response.errorBody());
                         Log.d(TAG, "ERROR: " + error.getDetail());
                     } catch (IOException e) {
                         Log.d(TAG, "EXCEPTION: " + e.getMessage());
```
Comment (kazik117) [inline Volontulo/app/src/main/java/com/stxnext/volontulo/ui/offers/AddOfferFragment.java]: It is good to describe what kind of error occured via `message` parameter and pass `e` as third parameter to logger method.

File: Volontulo/app/src/main/java/com/stxnext/volontulo/ui/offers/OfferEditFragment.java [hunk 2]
```diff
@@ -0,0 +1,375 @@
...
+import butterknife.BindView;
+import butterknife.OnClick;
+import io.realm.Realm;
+import okhttp3.ResponseBody;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Converter;
+import retrofit2.Response;
+
+public class OfferEditFragment extends VolontuloBaseFragment {
```
Comment (kazik117) [inline Volontulo/app/src/main/java/com/stxnext/volontulo/ui/offers/OfferEditFragment.java]: It is CP (aka copy-pasted) from `AddOfferFragment` or simply these two merged into one?

File: Volontulo/app/src/main/java/com/stxnext/volontulo/ui/offers/AddOfferFragment.java (hunk 1 above)
Comment (rdrewa) [inline Volontulo/app/src/main/java/com/stxnext/volontulo/ui/offers/AddOfferFragment.java]: The third parameter has Throwable type...

Comment (kazik117) [inline Volontulo/app/src/main/java/com/stxnext/volontulo/ui/offers/AddOfferFragment.java]: All exceptions in Java are Throwable.

**Pull request merged**